compliance/management/commands/decrypt_exports_inquiry.py

```diff
@@ -0,0 +1,76 @@
+"""
+Management command to decrypts a user's ExportInquiryLog record
+"""
+import sys
+
+from django.core.management import BaseCommand
+from django.contrib.auth import get_user_model
+from nacl.encoding import Base64Encoder
+from nacl.public import PrivateKey
+
+from compliance.api import decrypt_exports_inquiry
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    """
+    Management command to decrypts a user's ExportInquiryLog record
+    """
+
+    help = "Decrypts a user's ExportInquiryLog record"
+
+    def add_arguments(self, parser):
+        """
+        Definition of arguments this command accepts
+        """
+        group = parser.add_mutually_exclusive_group(required=True)
+        group.add_argument("--id")
+        group.add_argument("--email")
+        group.add_argument("--username")
+
+    def handle(self, *args, **options):
+        """Run the command"""
+
+        if options["id"]:
+            user = User.objects.get(id=options["id"])
+        elif options["username"]:
+            user = User.objects.get(username=options["username"])
+        elif options["email"]:
+            user = User.objects.get(email=options["email"])
+
+        if user is None:
+            self.stderr.write(self.style.ERROR("User doesn't exist "))
+            sys.exit(1)
+
+        log = user.exports_inquiries.last()
```

Comment:
It looks like there could be multiple `ExportsInquiryLog` entries per user, at least there are on RC. Which one should we pick? We should add an order clause here or in the model.

Reply:
There's a representation of what this is here:

https://github.com/mitodl/mitxpro/blob/master/authentication/pipeline/user.py#L211

so I've refactored that into an API method for that code and the command to both use.